packages/native/src/useScrollToTop.tsx

```diff
@@ -82,20 +83,23 @@ export default function useScrollToTop(
           const scrollable = getScrollableNode(ref);
 
           if (isFocused && isFirst && scrollable && !e.defaultPrevented) {
-            if ('scrollToTop' in scrollable) {
-              scrollable.scrollToTop();
-            } else if ('scrollTo' in scrollable) {
-              scrollable.scrollTo({ y: 0, animated: true });
+            if ('scrollTo' in scrollable) {
```

Comment:
I changed the order of these function calls so prioritizes scroll functions that support offset